intro.tex

```diff
@@ -1,55 +1,67 @@
 \section{Introduction}
 
-This  note describes the plan for ensuring the Rubin community will have the data products and services necessary to produce early science during the time between commissioning and the first data release in year 1. 
-The current planned start date for full survey operations is 1 April 2024, with considerable uncertainty. 
-As of November 2021, the construction project schedule shows completion between December 2023 and July 2024 (i.e. planned finish and finish with contingency). 
-It is unlikely the project will finish in December 2023 and highly likely it will finish by July 2024. 
+This note describes the plan for ensuring the Rubin community will have the data products and services necessary to produce high-impact early science during the time between commissioning and the first LSST Data Release, DR1.  
+Community expectations for early science are high due to the transformative nature of the Rubin data and the extensive amount of on-sky time planned in commissioning and science validation.
 
 \subsection{Definition of Early Science}  \label{ssec:defn}
 
 Early Science (ES) is defined as any science enabled by Rubin for its community through and including the first data release, \drone.
-DR1 will be based on the first 6 months of data and is scheduled for release 12 months  after the start of full survey operations, \citep{RDO-011}.
-Community expectations for early science are high due to the transformative nature of the Rubin data and the extensive amount of on-sky time planned in commissioning and science validation.
 
-\subsection{Motivations for an Early Science Program}
+DR1 is the first planned release of static-sky data products. 
```

Comment:
Is that true? What about DP0, DP1, and DP2? Those all include static-sky data products. Maybe this sentence needs to add "from LSST"

Reply:
"from the full LSST survey" -- readers may not appreciate Rubin v. LSST

Reply:
yes, from the full survey and in the baseline plan. This document describes the addition of data previews. I will add Robert's clarification be remove any ambiguity

Reply:
although - looks like I had already rewritten this part before I saw your comments.